drafts/draft-0005.md

```diff
@@ -0,0 +1,188 @@
+1. Before submitting the transaction, it should contain at least one valid signature.
+1. `POST` to the `MULTISIG_ENDPOINT` endpoint with the SEP-0007 `tx` URI in the body as the parameter `uri`.
+1. The service responds with a 200 HTTP status code and a JSON object specified in the Submit Transaction Endpoint Response section.
+
+## Rationale
+
+When implementing StellarGuard for the Stellar Account Viewer, it was rightly pointed out it should avoid vendor-specific implementation. This SEP provides an alternative approach that can be used by tools that want to support multisig accounts without implementing a multisig server for each product. 
+
+One design choice was whether to skip the `stellar.toml` step and add the fully qualified multisig server endpoint to the account's data field. The `stellar.toml` approach affords greater flexibilty for the server to change its implementations. It is much easier to update the `stellar.toml` than to update the data entry of every account that uses it.
+
+## Implementations
+
+The server implementation is currently live on [https://test.stellarguard.me](https://test.stellarguard.me) and [https://stellarguard.me](https://stellarguard.me).
+
+A library that implements this protocol has been published here: [https://github.com/stellarguard/multisig-utils](https://github.com/stellarguard/multisig-utils)
+
+A demo using that library that submits to the StellarGuard multisig endpoint: [https://stellarguard.github.io/multisig-utils/demo/](https://stellarguard.github.io/multisig-utils/demo/)
+
+## Outstanding Questions/Discussion
+
+1. Should we specify that multisig services expose and return a `statusHref` field for status updates in the response data and also dictate how that should endpoint should function? Applications may have specific use-cases or implementations that this may not work with (for instance, what if they want to use event source or websockets instead of a polling endpoint -- or if they do not want to expose a public status endpoint at all for privacy reasons). However, if we do not specify exactly how it should work then it's possible there will be multiple incompatible implementations.
+
+1. Do we need to explicitly define how the endpoints should look, such as adding `/submit` or `/status` to the MULTISIG_ENDPOINT?
+
+1. Should we use `MULTISIG_SERVER` or `MULTISIG_ENDPOINT` as the key in stellar.toml. `SERVER` is consistent with the naming of `FEDERATION_SERVER`, but `MULTISIG_ENDPOINT` more accurately describes that it is not just a hostname, it's the actual path to the endpoint.
+
+1. What should the data entry key be for the multisig domain? `multisig.domain` aligns with any particular "namespaces" implementation that may come out of [this discussion](https://github.com/stellar/stellar-protocol/pull/199). However `multisig_domain` is more consistent with the field `home_domain` used on the account.
```

Comment:
conf.multisig.domain is likely to be the thing.

Reply:
Just reiterating what we talked about privately: "config.multisig.domain" (looks like we have plenty of chars leftover in they key size so no need to abbreviate it).

Reply:
Ok we go this way.

Reply:
Resolved